test/celeritas/global/AlongStep.test.cc

```diff
@@ -426,6 +470,57 @@ TEST_F(SimpleCmsAlongStepTest, msc_field_finegrid)
         EXPECT_DOUBLE_EQ(0, result.alive);
     }
 }
+
+TEST_F(SimpleCmsRZFieldAlongStepTest, msc_rzfield)
+{
+    size_type num_tracks = 128;
+    Input inp;
+    {
+        // This track takes ~150k substeps in the field propagator before
+        // reaching a boundary.
+        SCOPED_TRACE("electron taking large step in vacuum");
+        inp.particle_id = this->particle()->find(pdg::electron());
+        inp.energy = MevEnergy{0.697421113579829943};
+        inp.phys_mfp = 0.0493641564748481393;
+        inp.position = {-33.3599681684743388, 1.43414625226707426, -700.000001};
+        inp.direction = {-0.680265923322200705,
+                         0.731921125057842015,
+                         -0.0391118941072485030};
+        // Step limited by distance to interaction = 2.49798914193346685e21
```

Comment:
Are these comments accurate? This looks like a copy-paste of the test above, which was testing errors @amandalund had spotted while running the demo loop. The second test below has identical results to the non-RZ field. I think for both cases you should test `result.displacement` as well as `result.angle`.